apps/api/src/app/auth/auth.controller.ts

```diff
@@ -59,13 +61,19 @@ export class AuthController {
   ) {}
 
   @Get('/github')
-  githubAuth() {
+  githubAuth(context: ExecutionContext) {
+    Logger.verbose('Checking Github Auth');
+    Logger.debug('Context: ' + context);
     if (!process.env.GITHUB_OAUTH_CLIENT_ID || !process.env.GITHUB_OAUTH_CLIENT_SECRET) {
-      throw new ApiException(
+      const err = new ApiException(
         'GitHub auth is not configured, please provide GITHUB_OAUTH_CLIENT_ID and GITHUB_OAUTH_CLIENT_SECRET as env variables'
       );
+      Logger.error(err);
+      throw err;
     }
 
+    Logger.log('Github Auth has all variables.');
```

Comment:
What is the difference between info and log?

Reply:
they are the same thing its just how the nest logger interface is.